tests/test_process.py

```diff
@@ -129,50 +129,30 @@ def test_get_lc2_path():
 
 @responses.activate
 def test_get_s2_metadata_not_found():
-    responses.add(responses.GET, f'{process.S2_SEARCH_URL}/foo', status=404)
-    responses.add(
-        responses.POST, process.S2_SEARCH_URL,
-        body='{"numberReturned": 0}', status=200,
-    )
+    responses.add(responses.GET, f'{process.S2_GRANULE_DIR}////foo.SAFE/manifest.safe', status=404)
     with pytest.raises(ValueError):
         process.get_s2_metadata('foo')
 
 
 @responses.activate
-def test_get_s2_metadata_cog_id():
-    responses.add(
-        responses.GET, f'{process.S2_SEARCH_URL}/2FS2B_22WEB_20200913_0_L2A',
-        body='{"foo": "bar"}', status=200,
-    )
-
-    assert process.get_s2_metadata('2FS2B_22WEB_20200913_0_L2A') == {'foo': 'bar'}
+def test_get_s2_metadata():
+    # TODO implement me
+    pass
 
 
-@responses.activate
-def test_get_s2_metadata_esa_id():
-    responses.add(
-        responses.GET, f'{process.S2_SEARCH_URL}/S2B_MSIL2A_20200913T151809_N0214_R068_T22WEB_20200913T180530',
-        status=404,
-    )
-    responses.add(
-        responses.POST, process.S2_SEARCH_URL,
-        body='{"numberReturned": 1, "features": [{"foo": "bar"}]}', status=200,
-    )
+def test_get_s2_path(test_data_directory):
+    scene_name = 'S2A_MSIL1C_20160616T112217_N0204_R137_T29QKF_20160617T193500'
+    f = open(f'{test_data_directory}/{scene_name}.manifest.safe', 'r')
+    manifest_text = f.read()
```

Comment:
It's typical in Python to use `with` along with `open` so the file is automatically closed as soon as you're finished with it, e.g.:
```suggestion
    with open(f'{test_data_directory}/{scene_name}.manifest.safe', 'r') as f:
        manifest_text = f.read()
```

More details at https://towardsdatascience.com/why-you-should-use-context-managers-in-python-4f10fe231206